src/main/xerus/monstercat/api/Player.kt

```diff
@@ -181,6 +205,10 @@ object Player: FadingHBox(true, targetHeight = 25) {
 			}
 			add(pauseButton.apply { isSelected = false })
 			add(stopButton)
+			add(buttonWithId("skipback") { playPrev() }).tooltip("Previous")
```

Comment:
I think this button should automatically be disabled when there is no previous track to avoid confusion in interaction.

Reply:
7841047d9ee9eddda63760896916bcc947d1f8ea